Cargo.toml

```diff
@@ -8,4 +8,13 @@ members = [
 ]
 exclude = [
     "packages/any",
-]
\ No newline at end of file
+]
+
+# [patch.crates-io]
+# wasmer = { path = "../wasmer/lib/api" }
+# wasmer-wasi = { path = "../wasmer/lib/wasi" }
+# wasmer-types = { path = "../wasmer/lib/types" }
+# wasmer-compiler-cranelift = { path = "../wasmer/lib/compiler-cranelift" }
+# wasmer-compiler-llvm = { path = "../wasmer/lib/compiler-llvm" }
+# wasmer-compiler-singlepass = { path = "../wasmer/lib/compiler-singlepass" }
+# wasmer-compiler = { path = "../wasmer/lib/compiler" }
```

Comment:
Don't commit this. Also undo the changes to Cargo.lock.

Reply:
The `Cargo.lock` changes I think I forgot to commit from `master`, so they were dangling (not part of this PR)